plugin/mysql.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""!
+    ____  ____  ______       __      __       __       _____
+   / __ )/ __ \/ ___/ |     / /___ _/ /______/ /_     |__  /
+  / __  / / / /\__ \| | /| / / __ `/ __/ ___/ __ \     /_ <
+ / /_/ / /_/ /___/ /| |/ |/ / /_/ / /_/ /__/ / / /   ___/ /
+/_____/\____//____/ |__/|__/\__,_/\__/\___/_/ /_/   /____/
+                German BOS Information Script
+                     by Bastian Schroll
+
+@file:        mysql.py
+@date:        15.02.2021
+@author:      Jan Speller
+@description: Mysql Plugin
+"""
+import logging
+from plugin.pluginBase import PluginBase
+
+# ###################### #
+# Custom plugin includes #
+import mysql.connector
+from datetime import datetime
+
+# ###################### #
+
+logging.debug("- %s loaded", __name__)
+
+
+class BoswatchPlugin(PluginBase):
+    """!Description of the Plugin"""
+
+    def __init__(self, config):
+        """!Do not change anything here!"""
+        super().__init__(__name__, config)  # you can access the config class on 'self.config'
+
+    def onLoad(self):
```

Comment:
Nur als Idee: 
- SQL als File irgendwo ablegen
- Im onLoad() prüfen ob die DB existiert, ansonsten direkt anlegen.

Dann würde händisches Anlegen entfallen. Was meinst du?

Reply:
Hab das Sql-File lokal schon mit drin, wusste nur noch nicht, wie ich dam am simpelsten löse,
das mit der onLoad is aber ne Geile Idee